src/evernodeone.c

```diff
@@ -67,6 +67,26 @@ int64_t hook(uint32_t reserved)
             int is_evr;
             IS_EVR(is_evr, amount_buffer, issuer_accid);
 
+            // Checking whether this host has an initiated transfer to continue.
+            int has_initiated_transfer = 0;
+            TRANSFEREE_ADDR_KEY(account_field);
+
+            uint8_t transferee_addr[TRANSFEREE_ADDR_VAL_SIZE];
+            uint8_t prev_host_addr[HOST_ADDR_VAL_SIZE];
+            uint8_t prev_token_id[TOKEN_ID_VAL_SIZE];
+            if (state(SBUF(transferee_addr), SBUF(STP_TRANSFEREE_ADDR)) != DOESNT_EXIST)
+            {
+                has_initiated_transfer = 1;
+
+                HOST_ADDR_KEY((uint8_t *)(transferee_addr + TRANSFER_HOST_ADDRESS_OFFSET));
```

Comment:
Feels like more readable if we move this `HOST_ADDR_KEY` and `TOKEN_ID_KEY` to later else block whether we check `has_initiated_transfer == 0`